.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Don't think this should be included?

Reply:
you can add this to your local global `~/.gitignore` file